sbncode/TPCReco/NuVertexChargeTree_module.cc

```diff
@@ -504,33 +648,57 @@ void sbn::NuVertexChargeTree::FillStubPFPs(
       fStubPFPTruePy.push_back(-1);
       fStubPFPTruePz.push_back(-1);
     }
+
   } // end iterate over Stub-PFParticle's
 }
 
 void sbn::NuVertexChargeTree::FillStubs(
-    const std::vector<art::Ptr<sbn::Stub>> &stubs,
+    const std::vector<sbn::StubInfo> &stubs,
     const std::vector<unsigned> &stub_hit_inds,
-    const std::vector<std::vector<art::Ptr<recob::Hit>>> &stub_hits,
+    const geo::GeometryCore *geo,
```

Comment:
For uniformity with the other services, I suggest `geo::GeometryCore const& geom` (i.e. a reference rather than a pointer): I am assuming here that the code will not work if `geom` is null.
Also, I suggest renaming `geo` to `geom`, since `geo` is already a namespace: the compiler seems to understand, but it's a bit confusing.

Reply:
Is there a problem with using a variable name that happens to line up with a namespace? As you said, it's acceptable for C++.

Also, the service function getter returns a pointer, so it's natural to use a pointer to carry "geo/geom" around. I don't think switching from a pointer would change any behavior of the code.